active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+- In a Desktop app, this method is called every time a Window object is created.
+
+## XamlWindowActivatedEventArgs class
+
+Contains the windows activation state information returned by the Window.Activated event.
+
+> Issue: should rename these to avoid collision.
+
+> Spec note: this is a new XAML version of the existing 
+Windows.UI.Core.[WindowActivatedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.WindowActivatedEventArgs).
+
+## XamlWindowActivatedEventArgs properties
+
+**Handled**
+
+Gets or sets whether the window activation changed event was handled.
+ **true** if the event has been handled by the appropriate delegate; **false** if it has not.
+
+```CS
+public bool Handled { get; set; }
+```
+
+**WindowActivationState**
+
+Gets the activation state of the window at 
+the time the event was raised.
+
+```CS
+public CoreWindowActivationState WindowActivationState { get; }
+```
+
+## XamlVisibilityChangedEventArgs class
+
+Contains the arguments returned by the event fired when a Window instance's visibility changes.
+
+> Spec note: this is a new Xaml version of the existing 
+Windows.UI.Core.[VisibilityChangedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.VisibilityChangedEventArgs).
+
+
+### XamlVisibilityChangedEventArgs
+
+Contains the arguments returned by the event fired when a Window instance's visibility changes.
+
+> Spec note: this is a new XAML version of the existing 
+Windows.UI.Core.[VisibilityChangedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.VisibilityChangedEventArgs).
+
+**Handled**
+
+Gets or sets whether the window activation changed event was handled. **true** 
+if the event has been handled by the appropriate delegate; **false** if it has not.
+
+**Visible**
+
+Gets whether the window is visible or not. **true** if the window is visible; otherwise is **false**.
+
+## XamlWindowSizeChangedEventArgs class
+
+> Spec note: this is a new XAML version of the existing 
+Windows.UI.Core.[WindowSizeChangedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.WindowSizeChangedEventArgs).
+
+Contains the argument returned by a window size change event.
+
+### XamlWindowSizeChangedEventArgs Properties
+
+**Handled**
+
+Gets or sets whether the  event was handled. **true** if the event has been handled 
+by the appropriate delegate; **false** if it has not.
+
+**Size**
+
+Gets the new size of the window in units of effective (view) pixels.
+
+
+
+## XamlLaunchActivatedEventArgs class
+
+Provides event information when an app is launched.
+
+> Spec note: this is a new XAML version of the existing Windows.ApplicationModel.Activation.
+[LaunchActivatedEventArgs](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Activation.LaunchActivatedEventArgs).
+
+## XamlLaunchActivatedEventArgs Properties
+
+**Arguments**
+
+Gets the arguments that are passed to the app during its launch. 
+In a UWP app, this is equivalent to LaunchActivatedEventArgs.Arguments.
```

Comment:
I assume that for a Desktop app, the Arguments is the raw command line that comes after the program name (leading spaces preserved)? I.e., we don't try to parse it in any way with CommandLineToArgvW or anything like that?